### DISCUSSION TITLE
Welcome to coldestgame Discussions!

### DISCUSSION THREAD

**Discussion by kwadroke** [kwadroke/coldestgame#9] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Ariemeth** [comment]
Hi.  Been years since I had even thought about coldestgame.  Nice to see development is starting back up.

**Reply by kwadroke** [reply]
Yeah. been going back and forth for a while on getting back to it. Finally decided to give it a go again.
Been hoping for another completely FLOSS mech game to come around, but doesn't seem to be any others. Closest is BattleMetal.

Planning on using it for a Raspberry Pi Based Cockpit for a Maker Faire a few hours drive away.

**Reply by Ariemeth** [reply]
That sounds exciting.
